extensions/wikia/Discussions/EnableDiscussionsController.class.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Email\Fatal;
+
+class EnableDiscussionsController extends \WikiaController {
+
+	const ENABLE_DISCUSSIONS_NAVIGATION = 'wgEnableDiscussionNavigation';
+	const ENABLE_DISCUSSIONS = 'wgEnableDiscussion';
+	const ENABLE_FORUMS = 'wgEnableForumExt';
+
+	const P_SITE_ID = 'siteId';
+	const REASON = 'Enabling discussions!';
+
+	public function index() {
+		$this->request->isInternal();
+
+		$siteId = $this->request->getParams()[self::P_SITE_ID];
+
+		if ( !is_numeric( $siteId ) || $siteId < 0 ) {
+			throw new Fatal( 'Please provide a valid siteId, provided ' . $siteId );
+		}
+
+		$discussions = $this->setVariable( $siteId, self::ENABLE_DISCUSSIONS, true );
```

Comment:
Is `GET` the correct HTTP verb for this? Maybe variables should change only on `POST` requests.